File: 1.Interview_practce/Array/Largest_in_arr.js


function largest(){
    const arr=[1,5,2,,66,3,2,55,5,552]
    let largest=0;
    for(let i=0;i<arr.length;i++){
        if(arr[i]>largest){
            largest=arr[i]
        }
    }
    console.log(largest)
}
largest()

File: 1.Interview_practce/Array/Largest_in_arr.java
public class Largest_in_arr {
    public static void main(String[] args) {
        int arr[]={1,5,2,3,66,44,55,22,220};
        int largest=0;

        for(int i=0;i<arr.length;i++){
            if(arr[i]>largest){
                largest=arr[i];
            }
        }
        System.out.println(largest);
    }
}
